src/core/inline-idl-parser.js

```diff
@@ -153,12 +154,15 @@ function renderInternalSlot(details) {
   const { identifier, parent, renderParent } = details;
   const { identifier: linkFor } = parent || {};
   const lt = `[[${identifier}]]`;
-  const html = hyperHTML`${parent && renderParent ? "." : ""}[[<a
+  const dot = parent && renderParent ? "." : "";
+  const anchor = hyperHTML`<a
     data-xref-type="attribute"
-    data-link-for=${linkFor}
```

Comment:
Soo.. it doesn't support not adding undefined/null attributes like hyperHTML? :disappointed: 
If not, should we raise this issue at nanohtml?